icssc/AntAlmanac
# Move screenshot button to calendar toolbar

## Summary
- Moved the screenshot button from Export to the calendar toolbar, as discussed in last week's meeting. The screenshot button did not work on mobile devices because on small screen sizes, the Export button moves into the Settings menu. Since the Settings menu is on top of the main page, we cannot properly take a screenshot of the calendar.
- To save more space in the toolbar, the button to add custom events is now just a regular icon button. The "Custom" label was removed.

## Test Plan
- Click the screenshot button on desktop and verify that the schedule was downloaded successfully.
- Click the screenshot button on mobile viewports and verify that the schedule was downloaded succesfully.

## Issues

Closes #979
